docs/blog/2017-03-08-gatsbygram-case-study/index.md

```diff
@@ -252,13 +193,13 @@ exports.createPages = ({ args }) => (
 )
 ```
 
-Template components themselves are again just plain React.js components.
-The optional context data you specify when creating pages is
-automatically passed in as a prop to the component. They're also passed
-as [GraphQL variables](http://graphql.org/learn/queries/#variables) so
-you can easily write dynamic queries for additional information.
+This allows us to create pages from any data, we just have to have the data accessible through our graphql schema first... more on that later.
+
+## Using templates
 
-This is what Gatsbygram's post template component looks like:
+Gatsby uses standard React.js components to render static pages. When you define a page in the `createPages` api, you also define a component to build it with. Those components, usually called templates get reused with different data to generate different pages. Usually, we will pass some simple information about what page we are on to the template component, and then the template will make a `graphql` query using that information to get the data it needs.
```

Comment:
Don't need commas in "Usually, we will..." sentence. That probably should be broken into two sentences remove the "and" in the middle.